server.go

```diff
@@ -127,6 +145,16 @@ func main() {
 	}
 	s.Engine.GET("/ScheduleHuddles", huddleController.ScheduleHandler)
 
+	if lfpath != "" {
+		glf, err := os.OpenFile(lfpath + "/gin.log", os.O_RDWR|os.O_APPEND, 0755)
+		if os.IsNotExist(err) {
+			glf, err = os.Create(lfpath + "/gin.log")
+		} else if err != nil {
```

Comment:
This needs to be a new `if` statement instead of an `if else`.  Otherwise the `else` (`s.Engine.Use(gin.LoggerWithWriter(glf))`) won't get called when `os.IsNotExist` (since it's all part of the same big if branch).